tests/flakes/flakes.sh

```diff
@@ -468,3 +512,243 @@ nix store delete $(nix store add-path $badFlakeDir)
 
 [[ $(nix path-info      $(nix store add-path $flake1Dir)) =~ flake1 ]]
 [[ $(nix path-info path:$(nix store add-path $flake1Dir)) =~ simple ]]
+  outputs = { self, nixpkgs, b }: {
```

Comment:
I love to see more test cases, but this doesn't look right. Did a rebase go wrong?

Reply:
fixed